TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousRight.java

```diff
@@ -611,66 +611,64 @@ private void autonomousCore() {
             readColorSensor(sleeveColor); // reading sleeve signal
             Logging.log("Autonomous - complete Sleeve color read.");
             // push sleeve cone out, and reading background color for calibration
-            robotRunToPosition(36.0, true);
+            robotRunToPosition(39.0, true);
         }
         parkingLocation = calculateParkingLocation(sleeveColor, backgroundColor);
         Logging.log("Autonomous - parking lot aisle location: %.2f", parkingLocation);
 
         // lift slider during strafe to high junction
         setSliderPosition(HIGH_JUNCTION_POS);
-        robotRunToPosition(-4.0, true); // get rid of sleeve cone
+        robotRunToPosition(-7.0, true); // get rid of sleeve cone
 
 
-        robotRunToPosition(-12.0, false); // strafe robot half mat to left side
-        waitSliderRun(); // make sure slider has been lifted.
+       // turn robot 45 degrees left
+        Orientation imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        rotate(45 - AngleUnit.DEGREES.normalize(imuAngles.firstAngle), AUTO_ROTATE_POWER);
 
```

Comment:
Add the second rotation to make sure it turns 45 degrees.

            imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
            rotate(45-AngleUnit.DEGREES.normalize(imuAngles.firstAngle), AUTO_ROTATE_POWER);